spec.html

```diff
@@ -21828,8 +21828,12 @@ <h1>Runtime Semantics: Evaluation</h1>
     </emu-clause>
   </emu-clause>
 
-  <emu-clause id="sec-with-statement">
+  <emu-clause id="sec-with-statement" legacy>
     <h1>The `with` Statement</h1>
+    <emu-note>
+      <p>It is recommended that the legacy `with` statement is not used in new ECMAScript code. Consider alternatives, such as <emu-xref href="#sec-destructuring-assignment" title></emu-xref>, which are permitted in both strict mode code and non-strict code.</p>
```

Comment:
```suggestion
      <p>It is recommended that the Legacy `with` statement is not used in new ECMAScript code. Consider alternatives, such as <emu-xref href="#sec-destructuring-assignment" title></emu-xref>, which are permitted in both strict mode code and non-strict code.</p>
```

We need to use a capital L for Legacy now for it to auto-link.

Reply:
```suggestion
      <p>It is recommended that the legacy `with` statement is not used in new ECMAScript code. Consider alternatives, such as <emu-xref href="#sec-destructuring-assignment" title></emu-xref>, which is permitted in both strict mode code and non-strict code.</p>
```

Reply:
@michaelficarra Thank you for the suggestions and letting me know the good news!

I've fixed the auto-link Legacy and removed the unneeded `title`. 👍

Instead of directly applying the suggested grammar change, I slightly reworded the sentence to avoid ambiguity in numerical agreement (i.e., "Consider alternatives … which are…" is both right and wrong, depending on whether we read the "such as" clause as subordinate). The sentence now reads:

> Consider alternatives that are permitted in both strict mode code and non-strict code, such as…"